app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -33,7 +33,7 @@ const Notes = () => {
 
   return (
     <>
-      <NotesMenu showMenu={showMenu}></NotesMenu>
+      <Menu showMenu={showMenu}></Menu>
```

Comment:
Use inline closing in cases where there are no children for the component.
```suggestion
<Menu showMenu={showMenu} />
```

Reply:
Fixed in https://github.com/nivetha-yadhavan/wheel/pull/10/